packages/in_app_purchase/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: in_app_purchase
 description: A Flutter plugin for in-app purchases. Exposes APIs for making in-app purchases through the App Store and Google Play.
 author:  Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/plugins/tree/master/packages/in_app_purchase
-version: 0.2.1+4
+version: 0.2.2
```

Comment:
If there is no public API change, we just need to update the patch, thus 0.2.1+5
I might have missed something though. Please point it out if so.

Reply:
The only public API change was adding the nullability return value.  The podspec change means this will be imported as a module, so modulemaps get generated, and CocoaPods does a lot of other magic.  I'm happy to change this to 0.2.1+5 though!